packages/core/src/StayInTouch/index.js

```diff
@@ -0,0 +1,153 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { Grid } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+import A from "../A";
+import RichTypography from "../RichTypography";
+
+const useStyles = makeStyles((theme) => ({
+  root: {},
+  title: {
+    textAlign: "center",
+    [theme.breakpoints.up("md")]: {
+      textAlign: "left",
+    },
+  },
+  icon: {
+    width: "1.375rem",
+    height: "1.375rem",
+    objectFit: "contain",
+  },
+  link: {
+    display: "inline-block",
+    padding: "0 0.625rem",
+  },
+  links: {
+    "& > a": {
+      display: "inline-block",
+      borderRight: "1px solid white",
+    },
+    "& > a:last-of-type": {
+      border: "none",
+    },
+    [theme.breakpoints.up("md")]: {
+      justifyContent: "flex-start",
+      width: "auto",
+    },
+  },
+  text: {},
+}));
+
+function StayInTouch({ options, support, socialMedia, title, ...props }) {
+  const classes = useStyles(props);
+
+  return (
+    <Grid
+      container
+      justify="flex-start"
+      alignItems="center"
```

Comment:
Since we are using a grid we should allow the user to set the justify and alignItems by themself